scripts/qa_crowbarsetup.sh

```diff
@@ -1102,56 +1102,56 @@ function onadmin_set_source_variables
         develcloud6)
             CLOUDSLE12DISTPATH=${want_cloud6_iso_path:='/ibs/Devel:/Cloud:/6/images/iso'}
             [ -n "$TESTHEAD" ] && CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6:/Staging/images/iso
-            CLOUDSLE12DISTISO=${want_cloud6_iso:="SUSE-OPENSTACK-CLOUD-6-$arch*Media1.iso"}
+            CLOUDSLE12DISTISO=${want_cloud6_iso:="SUSE-OPENSTACK-CLOUD-6-${arch}-Media1.iso"}
```

Comment:
http://download.nue.suse.com/ibs/Devel:/Cloud:/6/images/iso/SUSE-OPENSTACK-CLOUD-6-x86_64-Media1.iso exists, but http://download.nue.suse.com/ibs/Devel:/Cloud:/6:/Staging/images/iso/SUSE-OPENSTACK-CLOUD-6-x86_64-Media1.iso does not - do we want to fix this on the IBS side?